### ISSUE TITLE
Follow / unfollow hover effect text alignment

### ISSUE THREAD

**Issue by sandervankasteel** [devinhalladay/spotio#16]
The text alignment of the follow / unfollow button is off when you follow an artist / playlist and hover over the button.

![screen shot 2016-08-25 at 09 01 35](https://cloud.githubusercontent.com/assets/17492906/17959804/ee2e5764-6aa2-11e6-81fa-d08b143cd97d.png)
![screen shot 2016-08-25 at 09 01 31](https://cloud.githubusercontent.com/assets/17492906/17959805/ee392824-6aa2-11e6-8ad8-e5e7a2cf7917.png)

**Comment by devinhalladay** [comment]
Thanks!

**Comment by devinhalladay** [comment]
This looks related to #13 as well

**Comment by ColeTownsend** [comment]
Wondering if hard coding it on both elements will work better? I cannot find this puppy.

**Comment by devinhalladay** [comment]
Yeah I may try that, still not sure which elements to target though/how high to make the specificity so as to not break future updates that add new buttons with the same .b-button-add class (I think that's what the class is, can't check cause I'm in class right now lol)
